bom/pom.xml

```diff
@@ -24,7 +24,7 @@
         <!-- compile/runtime dependency versions, managed in the BOM -->
         <classmateVersion>1.0.0</classmateVersion>
         <ehcacheVersion>2.6.9</ehcacheVersion>
-        <infinispanVersion>8.0.2.Final</infinispanVersion>
+        <infinispanVersion>8.2.0.Final</infinispanVersion>
```

Comment:
How did you pick these versions? As per the [WF 10 POM](https://github.com/wildfly/wildfly/blob/10.0.0.Final/pom.xml#L150..L166), it's 5.2.12.Final for Narayana and 8.1.0.Final for ISPN.